agent/dockerclient/mobyiface/interface.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+// Package mobyiface contains an interface for moby matching the
+// subset used by the agent
+package mobyiface
+
+import (
+	"context"
+	"io"
+	"time"
+
+	"github.com/docker/docker/api/types"
```

Comment:
Can you change this to "github.com/moby/moby" instead, which means you need to modify the `Gopkg.lock` and few other files?

Reply:
I think "github.com/docker/docker", is the convention, if you take a look at moby's code, they all use "github.com/docker/docker".

Reply:
The examples in moby/moby client have [//import "github.com/docker/docker/client"]. In the documentation as well it says to: import "github.com/docker/docker/client". Is there a particular reason you think moby/moby is better?

Reply:
`// import "github.com/docker/docker/api/types/events"` is at the top of the files too so that Go uses that import path.  I think these should stay the way they are now.